README.adoc

```diff
@@ -1,53 +1,72 @@
-= devops-stack-module-template
+= devops-stack-module-velero
 // Document attributes to replace along the document
 // Here you can define variables for something that keeps repeating along the text
-:chart-version: 5.0.2
+:velero-chart-version: 5.0.2
 :original-repo-url: https://github.com/vmware-tanzu/helm-charts
 
-A https://devops-stack.io[DevOps Stack] module to deploy a backup solution based on Velero.
+A https://devops-stack.io[DevOps Stack] module to deploy a backup solution based on https://velero.io[Velero].
+
+This module add the ability to perform backups of a cluster. This includes all Kubernetes objects, although with the GitOps approach of the Devops Stack, it is not necessary to have a backup of all Kubernetes objects, as they are generally disposable and recreated by ArgoCD.
 
 The Velero chart used by this module is shipped in this repository as well, in order to avoid any unwanted behaviors caused by unsupported versions.
 
 [cols="1,1,1",options="autowidth,header"]
 |===
 |Current Chart Version |Original Repository |Default Values
-|*{chart-version}* |{original-repo-url}/tree/main/charts/velero[Chart] | https://artifacthub.io/packages/helm/vmware-tanzu/velero/4.1.3?modal=values[`values.yaml`]
+|*{velero-chart-version}* |{original-repo-url}/tree/main/charts/velero[Chart] | https://artifacthub.io/packages/helm/vmware-tanzu/velero/{velero-chart-version}?modal=values[`values.yaml`]
 |===
 
 == Usage
 
-_Here you will show examples of how to call this module on your code. Note that the `depends_on` needs to be adapted to your use case._
+=== Backup
+
+Here is an example of the module usage:
 
-_You can start by giving a simple example and then show some others with increasing complexity and/or for specific use cases._
+- A default backend in a AWS S3 bucket, accessed with a ServiceAccount
+- A schedule at 4 a.m, which uses EBS snapshots as backend for PVC
+- A schedule at 2 a.m, which uses Restic as backend for PVC
+- Prometheus monitoring with a Grafana dashboard
 
 [source,terraform]
 ----
 module "backup" {
-  # source           = "git::https://github.com/camptocamp/devops-stack-module-backup.git//eks?ref=grafana"
-  source           = "../module-backup/eks"
-  target_revision  = "grafana"
+  source           = "git::https://github.com/camptocamp/devops-stack-module-velero.git//eks"
   cluster_name     = local.cluster_name
   base_domain      = local.base_domain
   argocd_namespace = module.argocd_bootstrap.argocd_namespace
 
+  ``
   enable_monitoring_dashboard = true
 
+  # Automated backups can be specified.
+  #
+  # The included Resources is very important: if the resource `pods` is specified,
+  # then velero will not attempt to use the native snapshot feature of the object
+  # storage of the cloud provider (EBS snapshots), and will use the File System Backup, i.e
+  # Restic.
```

Comment:
I would put this in an admonition after the example. Maybe it's me, but I think this warning would be more "visible" within an important box.